backend/main.py

```diff
@@ -79,8 +79,8 @@ def HandleError(e):
   'updateMission': api_calls.UpdateMission,
   'createChatRoom': api_calls.AddChatRoom,
   'updateChatRoom': api_calls.UpdateChatRoom,
-  'addPlayerToChat': api_calls.AddPlayerToChat,
```

Comment:
The code needs transforming into add player to group; chats got a group and not a list of players. As such, this end point shouldn't be in the list and the backing code is sorta dead, waiting for transforming.